target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-wr30u.dtsi

```diff
@@ -1,252 +1,23 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
 /dts-v1/;
-#include <dt-bindings/input/input.h>
-#include <dt-bindings/gpio/gpio.h>
+#include "mt7981b-xiaomi-mi-router-common.dtsi"
 
-#include "mt7981.dtsi"
-
-/ {
-	aliases {
-		serial0 = &uart0;
-		led-boot = &led_system_yellow;
-		led-failsafe = &led_system_yellow;
-		led-running = &led_system_blue;
-		led-upgrade = &led_system_yellow;
-	};
-
-	chosen {
-		stdout-path = "serial0:115200n8";
-	};
-
-	memory {
-		reg = <0 0x40000000 0 0x10000000>;
-	};
-
-	gpio-keys {
-		compatible = "gpio-keys";
-
-		reset {
-			label = "reset";
-			gpios = <&pio 1 GPIO_ACTIVE_LOW>;
-			linux,code = <KEY_RESTART>;
-		};
-
-		mesh {
-			label = "mesh";
-			gpios = <&pio 0 GPIO_ACTIVE_LOW>;
-			linux,code = <BTN_9>;
-			linux,input-type = <EV_SW>;
-		};
-	};
-
-	leds {
-		compatible = "gpio-leds";
-
-		led_system_blue: system_blue {
-			label = "blue:system";
-			gpios = <&pio 9 GPIO_ACTIVE_LOW>;
-		};
-
-		led_system_yellow: system_yellow {
-			label = "yellow:system";
-			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
-		};
-
-		led_network_blue {
-			label = "blue:network";
-			gpios = <&pio 11 GPIO_ACTIVE_LOW>;
-		};
-
-		led_network_yellow {
-			label = "yellow:network";
-			gpios = <&pio 12 GPIO_ACTIVE_LOW>;
-		};
-	};
-};
-
-&eth {
-	status = "okay";
-
-	gmac0: mac@0 {
-		compatible = "mediatek,eth-mac";
-		reg = <0>;
-		phy-mode = "2500base-x";
-
-		nvmem-cells = <&macaddr_factory_4 (-1)>;
-		nvmem-cell-names = "mac-address";
-
-		fixed-link {
-			speed = <2500>;
-			full-duplex;
-			pause;
-		};
-	};
+&gmac0 {
+	nvmem-cell-names = "mac-address";
+	nvmem-cells = <&macaddr_factory_4 (-1)>;
 };
 
-&mdio_bus {
-	switch: switch@1f {
-		compatible = "mediatek,mt7531";
-		reg = <31>;
-		reset-gpios = <&pio 39 GPIO_ACTIVE_HIGH>;
-		interrupt-controller;
-		#interrupt-cells = <1>;
-		interrupt-parent = <&pio>;
-		interrupts = <38 IRQ_TYPE_LEVEL_HIGH>;
+&leds {
+	led_network_blue {
```

Comment:
Please follow the pattern for the led node names. https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/leds-gpio.yaml

Reply:
Done